databases/week2/hyf_lesson2_homework.sql

```diff
@@ -0,0 +1,93 @@
+-- Part 1: Working with tasks
+INSERT INTO
+  task(
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id
+  )
+VALUES
+  (
+    "studying",
+    "study with coding database",
+    "2022-02-07",
+    "2022-02-07",
+    "2022-06-01",
+    1
+  );
+UPDATE
+  task
+SET
+  title = "prepare for week3 database"
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  due_date = "2022-01-01"
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  status_id = 2
+WHERE
+  id = 36;
+UPDATE
+  task
+SET
+  status_id = 3
```

Comment:
its correct, but, is it possible to do this in another way? lets say I know there is a status called 'Done', is it possible to update it to 'Done' without knowing what is the id of the corresponding status in this case 'Done' ?